lib/quick_travel/price_quote.rb

```diff
@@ -0,0 +1,12 @@
+require 'quick_travel/adapter'
+
+module QuickTravel
+  class PriceQuote < Adapter
+    self.api_base = '/api/price_quotes'
+
+    def self.calculate(params)
+      response = post_and_validate("#{api_base}/calculate", params)
+      new(response)
+    end
+  end
+end
```

Comment:
It seems the 2 key benefits are so when you cat a file your command prompt is on a new line, and so unix recognises it as a text file (again avoiding warnings)